resources/views/alert.blade.php

```diff
@@ -17,7 +17,7 @@
             'type',
         ])
 }}>
-    <div class="alert-icon-wrapper alert-{{ $type }}-icon @if($large) alert-icon-large @endif">
+    <div class="alert-icon-wrapper alert-{{ $type }}-icon @if($large) alert-icon-large flex-shrink-0 @endif">
```

Comment:
this will only make it apply to `large` alerts

Reply:
![image](https://user-images.githubusercontent.com/35610748/130628659-549170e4-8f9d-4bdb-8197-50f4d6460b5a.png)
:sadup:

did you also test if `large` is still showing fine after your changes @leMaur ?

Reply:
> this will only make it apply to `large` alerts

🤔 why apply it to `large` only? we got a non square with 40x44px...